### ISSUE TITLE
Using modules against 17.1.6 controller causes Invalid Version error

### ISSUE THREAD

**Issue by imayobrown** [avinetworks/avi_ansible_modules#6]
Hi All,

During the course of testing the modules with our upgrade from 16.4.3 to 17.1.6 we found that since we previously didn't specify ```api_version``` our old code generated ```Invalid Version``` errors on module execution when run against the controller at version 17.1.6. After some digging we found that the version being supplied by the modules is "16.4". The lack of having major, minor, and patch numbers in the semantic version causes this error. When we explicitly set the ```api_version``` to either "16.4.3" or "17.1.6" the module calls execute successfully. Should the modules be injecting a more specific version than "16.4"?

We are using the most recent version of the avisdk and modules installed through ansible galaxy against a controller at version 17.1.6.

**Comment by grastogi23** [comment]
Yes, we had a situation with the legacy sdk utils which were using 16.4 instead of 16.4.4. The requirement for micro version came after we had released the utils. I will push out the change such that new versions of roles and module with this setting.

**Comment by grastogi23** [comment]
https://github.com/avinetworks/sdk/pull/404

**Comment by imayobrown** [comment]
Thanks!
